operators/src/engine/operator.rs

```diff
@@ -65,41 +64,80 @@ pub trait PlotOperator: CloneablePlotOperator + Send + Sync + std::fmt::Debug {
     }
 }
 
-pub type InitializedVectorOperator =
-    dyn InitializedOperator<VectorResultDescriptor, TypedVectorQueryProcessor>;
+pub trait InitializedRasterOperator: Send + Sync {
```

Comment:
Why do we need a trait here instead of a concrete type?

Reply:
it was always a trait. It was a type alias for a trait